cachalot/monkey_patch.py

```diff
@@ -133,9 +141,7 @@ def inner(cursor, sql, *args, **kwargs):
                     if isinstance(sql, bytes):
                         sql = sql.decode('utf-8')
                     sql = sql.lower()
-                    if 'update' in sql or 'insert' in sql or 'delete' in sql \
-                            or 'alter' in sql or 'create' in sql \
-                            or 'drop' in sql:
+                    if SQL_DATA_CHANGE_RE.search(sql):
```

Comment:
One could argue that this is missing a `is not None`, but what you wrote is logically equivalent and marginally faster.

<img width="665" alt="Capture d’écran 2022-02-24 à 14 34 15" src="https://user-images.githubusercontent.com/1119169/155534011-06cb7455-2f21-4816-89d1-a8b6b707fe5e.png">